datalad_next/iter_collections/tests/test_itergitworktree.py

```diff
@@ -0,0 +1,74 @@
+from pathlib import PurePath
```

Comment:
Required below to compare names of `GitWorkTreeItem`s on Windows.
Generally on Windows `PurePath()` will return an instance of `PureWindowsPath`, while the property `GitWorkTreeItem.name` has type `PurePosixPath`. Therefore we have to explicltly compare `GitWorkTreeItem.name` properties with `PurePosixPath`-instances to ensure that it works on Windows and Linux. (See also additional patches below).

```suggestion
from pathlib import (
    PurePath,
    PurePosixPath,
)
```